### ISSUE TITLE
Export with BOM for Excel

### ISSUE THREAD

**Issue by nosovk** [MartinSahlen/bq-utils#4]
Hi, I found issue that probably was related to yours bq-util
https://github.com/googleapis/google-cloud-go/issues/292
https://github.com/googleapis/google-cloud-go/issues/1299
Have you find any workaround? Or you just patch csv after download? @albertog

**Comment by MartinSahlen** [comment]
Can you clarify a bit more how this issue relates to this repository?

**Comment by nosovk** [comment]
Yep. @albertog forked that repo. Soon he started original thread #292 
I make a conclusion that it he was in process of adapting those tools to usage on windows.
I found original repo, and according to doc it allows to make bq export from CLI, I mean you may have the same problems with excel like we have.

**Comment by MartinSahlen** [comment]
In order to proceed here I think you would need to give me some concrete examples or use cases or steps to reproduce issues you are facing.
